cli/meta.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	"github.com/flynn/flynn/controller/types"
+)
+
+func init() {
+	register("meta", runMeta, `
+usage: flynn meta
+       flynn meta set <var>=<val>...
+       flynn meta unset <var>...
+
+Manage metadata for an application.
+
+Examples:
+
+	$ flynn meta
+	KEY  VALUE
+	foo  bar
+
+	$ flynn meta set foo=baz bar=qux
+
+	$ flynn meta
+	KEY  VALUE
+	foo  baz
+	bar  qux
+
+	$ flynn meta unset foo
+
+	$ flynn meta
+	KEY  VALUE
+	bar  qux
+`)
+}
+
+func runMeta(args *docopt.Args, client *controller.Client) error {
+	app, err := client.GetApp(mustApp())
+	if err != nil {
+		return err
+	}
+
+	if args.Bool["set"] {
+		return runMetaSet(app, args, client)
+	} else if args.Bool["unset"] {
+		return runMetaUnset(app, args, client)
+	} else {
+		return runMetaGet(app, args, client)
+	}
+}
+
+func runMetaGet(app *types.App, args *docopt.Args, client *controller.Client) error {
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, 0, 8, 2, '\t', 0)
+	fmt.Fprintf(w, "KEY\tVALUE\n")
+	for k, v := range app.Meta {
+		fmt.Fprintf(w, "%v\t%v\n", k, v)
+	}
+	w.Flush()
+	return nil
+}
```

Comment:
We have [a function for creating a standard tabwriter](https://github.com/flynn/flynn/blob/2a16a7f46eff77f2501bd3b13b4116537e9a8e48/cli/main.go#L273-L275) and we also have [a convenience function for writing to it](https://github.com/flynn/flynn/blob/2a16a7f46eff77f2501bd3b13b4116537e9a8e48/cli/main.go#L277-L286) (see [here](https://github.com/flynn/flynn/blob/master/cli/ps.go#L34-L46) for example usage).